src/main/resources/reference.conf

```diff
@@ -201,10 +237,10 @@ cassandra-query-journal {
   
   # How many events to fetch in one query (replay) and keep buffered until they
   # are delivered downstreams.
-  max-buffer-size = 100
+  max-buffer-size = 10000
   
   # The fetch size of the Cassandra select statement
-  fetch-size = 50
+  fetch-size = 0
```

Comment:
For read journal we have proper backpressure all the way from the consumer of the stream, and we should take advantage of that and don't pull in too much into memory buffers. The drawback is of increased latency and not as high speed as would be possible with larger buffers. However, many concurrent readers could overload things, especially with large buffers. It is not possible to find a default that is perfect for all applications. 50 might be too conservative, perhaps increase somewhat?

For the PersistentActor replay we don't have this backpressure all the way, so the events will queue up in the mailbox if the actor is not fast enough to handle the replayed events. Handling replay events should be a quick action, so I'm pretty sure that the bottleneck will be earlier (e.g. deserialization). Using a larger fetch-size for those replays makes sense.